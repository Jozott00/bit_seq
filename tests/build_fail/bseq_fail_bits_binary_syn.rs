use bit_seq::bseq;

fn main() { bseq!(0b101) }