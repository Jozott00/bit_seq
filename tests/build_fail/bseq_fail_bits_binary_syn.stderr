error: expected bitsequence but got integer instead.
 --> tests/build_fail/bseq_fail_bits_binary_syn.rs:3:19
  |
3 | fn main() { bseq!(0b101) }
  |                   ^^^^^
